src/genie/libs/parser/hvrp/display_config_dhcp.py

```diff
@@ -0,0 +1,182 @@
+import logging
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+import re
+
+
+class DisplayConfigDhcpSchema(MetaParser):
+    schema = {
+        Any(): {
+            Optional("domain"): str,
+            Optional("gateway"): str,
+            Optional("netbios_servers"): list,
+            Optional("dns_servers"): list,
+            Optional("dhcp_excludes"): {
+                Any(): {
+                    Optional('end'): str,
+                    Optional('start'): str,
+                }
+            },
+            Optional("networks"): {
+                Any(): {
+                    Optional('ip'): str,
+                    Optional('subnet_mask'): str,
+                }
+            },
+            Optional("dhcp_options"): {
+                Any(): {
+                    Optional("option"): str,
+                    Optional("type"): str,
+                    Optional("data"): str,
+                }
+            },
+            Optional("lease_time"): str,
+        },
+    }
```

Comment:
Schemas are used to validate the key and value type of the output to ensure it's consistent. For that reason the topmost level key should never be an Any(). It's also best to not have keys set as Optional() unnecessarily. It's best to make every key possible be mandatory.